Header header
float64 goal_dist
int64 have_collide